### PR TITLE
feat(bindings/ffi): Add account data.

### PR DESCRIPTION
**Pull request by pixlwave**
Exposes the account data from #964 through the FFI.

### CODE REVIEW

File: bindings/matrix-sdk-ffi/src/client.rs [hunk 1]
```diff
@@ -223,6 +224,25 @@ impl Client {
         Ok(device_id.to_string())
     }
 
+    /// Get an event of the given type out of the account data store. The event
+    /// will be returned as a JSON string.
+    pub fn get_account_data_event(&self, event_type: String) -> anyhow::Result<Option<String>> {
+        RUNTIME.block_on(async move {
+            let event = self.store().get_account_data_event(event_type.into()).await?;
```
Comment (pixlwave) [inline bindings/matrix-sdk-ffi/src/client.rs:231]: I added this before `get_account_data_raw` was added to the account. Can update to use that method instead if it makes more sense.

Comment (jplatte) [reply bindings/matrix-sdk-ffi/src/client.rs:231]: I think it does, yeah.

Comment (pixlwave) [reply bindings/matrix-sdk-ffi/src/client.rs]: Updated (and slightly renamed the method).

While we're looking at this one, would there be a way for this method to return only the value of `content` as the JSON string? I've made the iOS `ClientProxy` decode the whole event and then return just the content but would be nice to do this in the SDK for all clients if it's somehow possible.

**Review by jplatte — APPROVED** [review]
Looks good. Please don't merge before my PR though.

File: bindings/matrix-sdk-ffi/src/client.rs (hunk 1 above)
Comment (jplatte) [reply bindings/matrix-sdk-ffi/src/client.rs]: Yeah, that would be possible and I really like the idea. I'll look into it tomorrow.

**Comment by pixlwave** [comment]
Sure thing, I'll leave merging to the Rust team for when everything is ready 🙂

**Comment by jplatte** [comment]
Now that half the changes are from me, wouldn't hurt if sbd. else from the team also reviews.

**Review by poljar — APPROVED** [review]

**Pull request merged**